pkg/tiller/release_install.go

```diff
@@ -162,7 +162,9 @@ func (s *ReleaseServer) performRelease(r *release.Release, req *services.Install
 
 		// update old release status
 		old.Info.Status.Code = release.Status_SUPERSEDED
-		s.recordRelease(old, true)
+		if err := s.recordRelease(old, true); err != nil {
+			return res, fmt.Errorf("Failed to update existing Release v%d status. Aborting install, no resources were updated", old.Version)
```

Comment:
This silences the error that `s.recordRelease` provides. Can we bubble that error up here using the following?

```
return res, fmt.Errorf("Release %q v%d failed to update: %v", old.Name, old.Version, err)
```

The message about "Aborting install, no resources were updated" is definitely useful, but that should be raised at the client level rather than the server side. I could see a use case where clients would want to handle this error and perform retries or graceful recoveries, for example. `cmd/helm/install.go` would be a good place to put it.